### DISCUSSION TITLE
the death of classic projects

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [github/gh-projects#45] [General]
Give them back. I hate the new projects. the new style totally buggers my process.

**Comment by mntlty** [comment]
sorry to hear that - this repo is dedicate to the cli extension to manage GitHub Projects. If you have general feedback on GitHub Projects, https://github.com/community/community/discussions/categories/issues is a more constructive place to share that.

**Comment by zer0Kerbal** [comment]
apologies - how about adding --repo to the choices (--user --org)?

**Reply by mntlty** [reply]
GitHub Projects aren't "owned" by a repo the same way that Projects Classics are - they are owned by a User or an Organization, and can be linked to a repo. https://github.com/cli/cli/releases/tag/v2.22.0 introduced some project commands for issues / prs that are scoped to a repo.
